client/src/templates/Challenges/redux/create-question-epic.js

```diff
@@ -43,13 +43,37 @@ function createQuestionEpic(action$, state$, { window }) {
     ofType(actionTypes.createQuestion),
     tap(() => {
       const state = state$.value;
-      const challengeFiles = challengeFilesSelector(state);
+      let challengeFiles = challengeFilesSelector(state);
       const {
         title: challengeTitle,
         superBlock,
         block,
         helpCategory
       } = challengeMetaSelector(state);
+      if (
+        challengeFiles?.some(file => file.editableRegionBoundaries.length > 0)
+      ) {
+        const editableRegionStrings = fileExtension => {
+          const startComment = fileExtension === 'html' ? '<!--' : '/*';
+          const endComment = fileExtension === 'html' ? '-->' : '*/';
+          return `\n${startComment} 'User Editable Region' ${endComment}\n`;
+        };
+
+        const filesWithEditableRegions = challengeFiles.map(file => {
+          const { contents: Contents, editableRegionBoundaries, ext } = file;
```

Comment:
```suggestion
          const { contents, editableRegionBoundaries, ext } = file;
```
This isn't necessary, we can just use `contents` as is.  Also, we typically use camelCase for variables and PascalCase for classes.